python/py-fmf/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        teemtee fmf 1.4.1
+
+name                py-fmf
+version             1.4.1
+revision            0
+
+categories-append   devel
+supported_archs     noarch
+license             GPL-2
+maintainers         {sub-pop.net:link @subpop} openmaintainer
+
+description         Flexible Metadata Format
+long_description    {*}${description}
+
+homepage            https://github.com/psss/fmf
+
+checksums           rmd160  01eb45c48f576e622caae6e03329183348103458 \
+                    sha256  4b4ca21ec1d028cff5c63b88dec6b73395b3dd68a0d724e6b1615b1719d0fc22 \
+                    size    86421
+
+python.versions     39 310 311 312
```

Comment:
Is there a strong reason to support old Python versions? Our default is now 3.12. @reneeotten may also have an opinion here.

Reply:
No strong opinion from me, other than these were the versions listed on the fmf pypi page.

Reply:
I'm not sure how to interpret @reneeotten 's thumbs-up response. Is it preferred to drop the older versions and only include 3.12?

Reply:
remove all except 312